### PR TITLE
Add Danish translation of the open letter

### PR DESCRIPTION
**Pull request by LunaTheFoxgirl**
I haven't translated the appendix at current time, but I may later.

Also I'd love to get some more eyes on the translation to make sure I didn't miss something.
(Do note that google translate fails pretty terrible at Danish translation so I'd recommend against using that to verify translation accuracy in general)

### CODE REVIEW

**Comment by combacsa** [comment]
If I may, could you apply #2536 as well?

File: index.da.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Et åbent brev for at fjerne Richard M. Stallman fra alle lederskabspositioner
+description: Vi, de underskrevende, vil have at hele FSF lederskabet at træde af og for Richard M. Stallman at blive fjernet fra alle lederskabspositioner. Vi vil ikke længere acceptere hans opførsel, lederskabsrolle og på anden vis holde hans skadefulde og farlige ideologi som acceptabelt.
+image: /assets/og-image.png
+twitter:
+    card: summary_large_image
+share_text: Jeg støtter dette kald for at hele FSF-bestyrelsen skal træde af og for at Richard M. Stallman skal fjernes fra alle lederskabspositioner.
+---
+
```
Comment (ehashman) [inline index.da.md:9]: Per https://github.com/rms-open-letter/rms-open-letter.github.io/pull/2545#issuecomment-810722881 the share links won't be generated without this fix :)

**Review by ehashman — COMMENTED** [review]
I put out a call for a community reviewer but I'll do a quick machine translation check and ship this if no response.

**Comment by LunaTheFoxgirl** [comment]
Will add share links once I'm at my pc. It's pretty early in the morning in Denmark atm.

**Comment by LunaTheFoxgirl** [comment]
I've added the share links fix @ehashman

File: index.da.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Der har været nok tolerance af RMSes modbydgelige idéer og opførsel. Vi kan ikke blive ved med at fortsætte at lade en person ruinere meningen i vores arbejde. Vores fællesskaber har ingen plads til folk som Richard M. Stallman og vi vil ikke fortsætte med at acceptere hans opførsel, give ham en lederskabsrolle, eller på nogen anden måde holde hans skadefulde og farlige ideologi som acceptabelt.
+
+**Vi vil have at hele FSF-bestyrelsen træder af.** Disse folk som har tilladt og givet RMS magt i mange år. De demonstrerer dette igen med at tillade ham at blive genindsadt i FSF-bestyrelsen. Det er tid til at RMS træder af fra fi software, tech etik, digitale rettigheder, og tech-fællsskaber, for han kan ikke give os det lederskab vi har brug for. **Vi vil også have at Richard M. Stallman bliver fjernet fra ALLE lederskabspositioner, inklusiv GNU-projektet.**
```
Comment (ehashman) [inline index.da.md:20]:
```suggestion
**Vi vil have at hele FSF-bestyrelsen træder af.** Disse folk som har tilladt og givet RMS magt i mange år. De demonstrerer dette igen med at tillade ham at blive genindsadt i FSF-bestyrelsen. Det er tid til at RMS træder af fra fri software, tech etik, digitale rettigheder, og tech-fællsskaber, for han kan ikke give os det lederskab vi har brug for. **Vi vil også have at Richard M. Stallman bliver fjernet fra ALLE lederskabspositioner, inklusiv GNU-projektet.**
```

Is this a typo?

**Review by ehashman — COMMENTED** [review]
Otherwise looks okay to me.

File: index.da.md (hunk 2 above)
Comment (LunaTheFoxgirl) [reply index.da.md:20]: Yes, that is a typo.

**Comment by LunaTheFoxgirl** [comment]
I just realized an other wording mistake, would it be okay for me to make a new PR with a fix for that?

**Comment by ehashman** [comment]
Of course!

**Pull request merged**